config/enabled.yml

```diff
@@ -1157,6 +1157,10 @@ Rails/PluralizationGrammar:
   Description: 'Checks for incorrect grammar when using methods like `3.day.ago`.'
   Enabled: true
 
+Rails/Presence:
+  Description: 'Checks code that can be written more easily using `Object#presence` defined by ActiveSupport.'
```

Comment:
Rails component better to be written as "Active Support" rather than "ActiveSupport".

> The proper names of Rails components have a space in between the words, like "Active Support". 

- https://github.com/rails/rails/blob/v5.2.0.beta2/guides/source/api_documentation_guidelines.md#wording
- https://github.com/rails/rails/tree/v5.2.0.beta2/activesupport

Reply:
I guess we should check this across the codebase.

Reply:
Surely.
But the proper name depends on the context so that Rails component is "Active Support" and Ruby module is "ActiveSupport". We will need to scrutinize it when checking the document.